### ISSUE TITLE
sourceTitle showing incorrect value via API

### ISSUE THREAD

**Issue by moverperfect** [Radarr/Radarr#2978]
**Describe the bug**
API response from /history shows incorrect "sourceTitle"

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to /api/history
2. View the "sourceTitle" attribute

**Expected behavior**
I expect sourceTitle attribute to return the source title of the downloaded resource e.g. "Minions 2015 720p BluRay DD-EX x264-xxxxxxxxxx" as shown in the wiki and also as returned in the sonarr implementation of this. Instead, I see the full name of the movie  e.g. "Minions 2015". I am using deluge as the client.

**Screenshots**
Deluge: ![image](https://user-images.githubusercontent.com/1750302/44538739-2950dc00-a6fa-11e8-9440-fd2b9b726ea1.png)
Radarr Api Response: ![image](https://user-images.githubusercontent.com/1750302/44538753-35d53480-a6fa-11e8-87e0-881930050ba0.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Mono Version: 5.4.0.167
 - Browser and Version [e.g. chrome, safari] (Only needed for UI issues)
 - Version 0.2.0.1120

**Debug Logs**
[radarr.debug.txt](https://github.com/Radarr/Radarr/files/2315372/radarr.debug.txt)

**Comment by galli-leo** [comment]
Is that for an imported movie history? Or the downloaded history? Can you post the full json output?

**Comment by moverperfect** [comment]
Hi @galli-leo 

Just done a couple hours of investigation searching through the code base(never looked at it before so was a bit confusing), started off at the end of the chain at the api, ended up getting to the DB, saw that it was wrong in the db. Went all the way back to when the event gets logged in the system. Couldn't see any glaring issues so I did a white box test from import start till when it gets logged in the DB, and the issue didn't replicate. I saw what I thought might have caused the issue which was line 160 in HistoryService.cs. Loaded up github and looked through the history, and it looks like you actually fixed the issue on August 5th with 77f146b262c5678adc3a56c01481800cb9c2c745 so thank you very much.

Tl;dr Issue fixed with 77f146b262c5678adc3a56c01481800cb9c2c745 on 5th Aug. When is this due to be pushed to users?

**Comment by galli-leo** [comment]
@moverperfect Thanks for digging! Yeah seems I fixed that :). Hopefully over the next few weeks. I only need to fix some small bugs and make sure the new release works ok.
